img/style.css

```diff
@@ -779,7 +779,6 @@ li.lms-ui-suggestion-item a {
 .lms-ui-button:hover {
   color: #336600;
   cursor: pointer;
-  text-decoration: underline;
```

Comment:
To jest poprawka nie w tym miejscu co trzeba - już poprawiłem gdzie indziej.

Reply:
Usunąłem.

Reply:
Ale teraz tu się pojawiło, a przecież w ogóle tej zmiany nie powinno być? ;-)